cfme/tests/control/test_basic.py

```diff
@@ -44,17 +45,181 @@
     # Needs working input/select mutability
 ]
 
-CONTAINER_POLICIES = [
+POLICIES = [
+    explorer.HostCompliancePolicy,
+    explorer.VMCompliancePolicy,
     explorer.ReplicatorCompliancePolicy,
     explorer.PodCompliancePolicy,
     explorer.ContainerNodeCompliancePolicy,
     explorer.ContainerImageCompliancePolicy,
+    explorer.HostControlPolicy,
+    explorer.VMControlPolicy,
     explorer.ReplicatorControlPolicy,
     explorer.PodControlPolicy,
     explorer.ContainerNodeControlPolicy,
     explorer.ContainerImageControlPolicy
 ]
 
+CONTROL_POLICIES = [
+    explorer.HostControlPolicy,
+    explorer.VMControlPolicy,
+    explorer.ReplicatorControlPolicy,
+    explorer.PodControlPolicy,
+    explorer.ContainerNodeControlPolicy,
+    explorer.ContainerImageControlPolicy
+]
```

Comment:
I was wondering, why this split between POLICIES and CONTROL_POLICIES? If they need to be splitted, can one be included in the other? (POLICIES = CONTROL_POLICIES + [ ... ] )